sdks/python/apache_beam/options/pipeline_options_validator_test.py

```diff
@@ -613,31 +615,29 @@ def test_environment_options(self):
         {
             'options': [
                 '--environment_type=pRoCeSs',
-                '--env_process_command=foo',
+                '--environment_options=process_command=foo',
                 '--environment_config=foo'
             ],
             'errors': ['environment_config']
         },
         {
             'options': [
                 '--environment_type=pRoCeSs',
-                '--env_process_command=foo',
-                '--env_process_variables=foo=bar',
-                '--env_docker_container_image=foo',
-                '--env_external_service_address=foo'
+                '--environment_options=process_command=foo',
+                '--environment_options=process_variables=foo=bar',
+                '--environment_options=docker_container_image=foo',
+                '--environment_options=external_service_address=foo'
```

Comment:
We could think about dropping the `env_` prefix for the options, e.g. `container_image` instead of `docker_container_image`.

Reply:
I want to keep the prefix as a hint for users. This way it is immediately obvious which environment type each option corresponds to.

Reply:
Sounds fair.